### DISCUSSION TITLE
Can't get one band member working on Jamulus

### DISCUSSION THREAD

**Discussion by klepto84** [jamulussoftware/jamulus#1436] [Technical Support & Help]
Three members of my bluegrass/folk group have been successful using Jamulus, but the fourth member experiences echo and can't sync with us. He uses an Elesis MultiMix8 and the rest of us have Focusrite Scarlett 2i2's. He tried a Scarlett and had the same problem. He has used different PCs. Replaced all of his cables, and this week changed ISP's and has 50g speed. He has tried different mics. He set up a server on his computer. No improvement.  We are all in the same city. 
At this point, we are out of things to check. Suggestions would be very much appreciated!

**Comment by JoeSouthern** [comment]
Swap him out with someone different? Ha ha - just joking. He's probably old like me (:

It appears the Alesis (I assume that's the one you mean) MulitMix8 is old - decade or better. I looked and they didn't have updated software - product is discontinued. It didn't appear they ever offered ASIO drivers - they did a funny wrap around, but I may be wrong. I realize you said he swapped out to a Scarlett - but I wonder if that route was given up too quickly or wasn't fully explored. Focusrite does have new asio drivers.

I read a little bit from things years ago - couldn't find recent things on the MulitMix8 and there were some issues with it playing nice with PC's then and this is now 10 years later. My Presonus mixer is probably a decade old, but they do offer recent ASIO drivers that work just fine for me. So, it's not just age - it's also the software.

The ASIO4All - from what I've seen - the preference is to use native ASIO drivers from the manufacturer - as in from Focusrite. If he swapped everything and still used ASIO4All - or an older buggy ASIO from MultiMix was still in there and his mixer was still hooked up ....well... I could see why the Focusrite didn't work. Could he try it with just the Focusrite and native driver - and no mixer in it at all?

Perhaps others are using it fine - I'm just taking a stab and he could replace it pretty cheaply.

After the driver, then what about his router? Is it old too? Perhaps there is an issue with it that has carried across all his attempted changes. I don't know specifically what might cause this in the router but it could be - hard knowing the settings on that router. I assume only one router.

You probably have to look at his machine too - any antivirus running weird things? Is it updated - machine and anti virus software? Any other process running on it that might interfere? Has he run a virus check?

I think on the speed - you could get good speed but have latency issues. On some it may take say 100ms to signal initially but then after the attention it will download at 300 speed. But, the latency might not be fast initially - think Starlink for example. I'm signed up for the beta for it and it's fast, but I'm not sure the latency will work for Jamulus because of what I just pointed out. Perhaps someone else has experience.

When he changed PC's - I assume they are Windows 10 - relatively recent and fast machines?

I assume you are all using the exact same version of Jamulus? Shouldn't matter if they are a little off on version numbers. But, is his a recent version?

I think you've got a couple of things - one is that when you've tested everything and it didn't work, you may need to retest each thing more thoroughly - or - let's hope not - that it's a combination of things you tried individually.

Is he hearing the echo from everyone or just himself? I've read where some have the mute on for themselves to fight the echo. I've had that on high latency connections. Not sure if that fixes it - just what I read.

It's possible he has some software setup on his machine causing a loopback that adds delay. Especially using a mixer. I use a mixer too. It took me awhile to get it set up right and I'm not sure it's perfect now but seems to be working. I assume there's no other AD (analog to digital) converter - other than the mixer - in the stream from his mixer to the outside world? Any other instrument software like IK Multimedia? I use that and it seems fine - but it's a for instance. Anything?

What does he show for his latency before he connects - from the Connection Dialog in Jamulus.

Also, I assume it's a wired connection not wifi and his isp isn't a cellular isp.

He could try from the command prompt a command - 

Tracert XXXXXX

Where XXXXX is the IP address of the server. It will show the loop times - maybe it will highlight something. It does 3 tries for each stop along the way to get an average trip time.

Unfortunately, I'm relatively new, so my suggestions are limited. Hope it gives some ideas.

**Reply by gene96817** [reply]
@JoeSouthern  covered a lot of topics. All valid places for problems. I prefer to start from either the client or the network and troubleshoot the chain by order of the data flow. Then there won't be confusion by looping from one issue to another. From this discussion, I suggest starting with the device and driver. Clean that up before going to the next item in the chain.

**Comment by klepto84** [comment]
Done that with a new Scarlett and new install of the native driver. We
tried using the Scarlett because 3 of us had success with the Scarlett. No
change from the Elesis.
After he upgraded to much better internet service with Bell, we lent him a
Scarlett and he reinstalled the driver, No improvement.

**Reply by gene96817** [reply]
well, three people successful with the Scarlett. Are you all on Windows? Which version? Can you say it is not a Jamulus issue?

When you report latency, do you mean ping time or overall delay? 
How are these numbers different for the four band members?
Where in the world are you? Are you all using the same ISP? Which network (ISP) is the server using?
The details matter.

**Comment by klepto84** [comment]
His ping time is 0 and overall delay is 28-38 ms. We have tried using a
server set up on his Windows 10 PC. His numbers are very similar to our
numbers., now
Initially, his ping times were over 20 and delay running over 150, but he
changed isp's and has much faster Bell Fibe internet, which is what I use.
He had a modem/router installed replacing the previous equipment
I gave him ethernet cable that I tested successfully with my set up.
He hears an echo which is better but still not bad when we play and
although three of us are in sync he perceives us as slowing down. I play
bass and use a metronome and I am bang on and the other guys follow me.
We are all in Kitchener-Waterloo Ontario and live a few miles from each
other.
He tried twice using a Scarlett 2i2 instead of his Elesis Multimix 8 with
no improvement. His ISP change is the only thing that has shown
improvement, but not enough to be functional.  Three of us are on Bell and
one is on Rogers--not the guy with the problem. He is on Bell.

**Comment by klepto84** [comment]
We have tried two servers, both with Bell and he has tried a number of
other public servers.

**Reply by gene96817** [reply]
Just trying a bunch of stuff assumes you are going to be lucky to eliminate the cause of the problem. Maybe you get lucky, maybe not. You might have to try a lot of stuff to get lucky.

**Comment by klepto84** [comment]
We've tried a lot of stuff and haven't gotten lucky yet.
Strangely the three connected players were pretty much plug and play.
